backend/src/test/resources/tests/sql/form/RELATIVE/QUARTERS_AFTER/REL_EXPORT_FORM.json

```diff
@@ -36,10 +36,24 @@
   ],
   "content": {
     "tables": [
-      "/shared/vers_stamm.table.json"
+      "/shared/vers_stamm.table.json",
+      "/shared/persons.table.json"
     ],
     "previousQueryResults": [
       "tests/form/EXPORT_FORM/RELATIVE/SIMPLE/query_results_1.csv"
     ]
+  },
+  "idColumns": {
```

Comment:
willst du das nicht lieber in die defaults der FormTests stecken?

Reply:
Gute Idee, aber die `persons`-Table müsste ich dann trotzdem in jedem Test angeben. Ich überlege, ob ich das auch per default im SQL Modus vom `SqlTestDataImporter` importieren lasse.

Reply:
es gibt ja schon dieses shared Zeug, eine Tabelle die 20PIDs und entsprechende Mappings enthält schadet sicher nicht